pom.xml

```diff
@@ -289,6 +288,11 @@
 				<artifactId>guice</artifactId>
 				<version>4.1.0</version>
 			</dependency>
+			<dependency>
+				<groupId>commons-lang</groupId>
+				<artifactId>commons-lang</artifactId>
+				<version>2.6</version>
```

Comment:
According to https://mvnrepository.com/artifact/commons-lang/commons-lang

The lastest version of commons-lang has been released 10 years.

Why not use https://mvnrepository.com/artifact/org.apache.commons/commons-lang3,
are there some compatible problem?

Reply:
It seems in a `dependencyManagement` not in a `dependency`, no problem...

Reply:
Right, it was included in platform bom but we removed the bom, so we need to add it here in dependencyManagement.